2.13/tests/test/test.js

```diff
@@ -473,10 +447,55 @@ function testDeprovision(instanceId, queryStrings, isAsync){
                     done();
                 })
             });
+
+            if (isAsync) {
+                // TODO: the query string should contain 'operation' for the last operation. FYI: https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md#parameters.
+                describe('DEPROVISION - poll', function() {
+                    it ('should return succeeded operation status after deprovision', function(done){
+                        pollInstanceLastOperationStatus(instanceId, done);
+                    })
+                });
+            }
         });
     });
 }
 
+function pollInstanceLastOperationStatus(instanceId, done) {
+    var count = 0;
+    var lastOperationState = 'in progress';
+    async.whilst(
+        function() {
+            return lastOperationState == "in progress" && count < config.maxPollingNum;
+        },
+        function(callback) {
+            count++;
+            preparedRequest()
+                .get('/v2/service_instances/' + instanceId + '/last_operation')
+                .set('X-Broker-API-Version', apiVersion)
+                .auth(config.user, config.password)
+                .expect(200)
+                .expect('Content-Type', /json/)
+                .end(function(err, res){
+                    if (err) return done(err);
+                    var message = validateJsonSchema(res.body, lastOperationSchema);
+                    if (message != "") return done(new Error(message));

```

Comment:
Sorry I didn't review it carefully... Also, here and line 479 `done` should not be called. Just `callback(err)` and leave the `done` calling to the result handling function below.

Reply:
Ok, thanks!